packages/zapp/console/src/components/WorkflowGraph/__stories__/WorkflowGraph.stories.tsx

```diff
@@ -49,7 +49,8 @@ stories.addDecorator((story) => (
 stories.add('TaskNodeRenderer', () => (
   <WorkflowGraph
     onNodeSelectionChanged={onNodeSelectionChanged}
-    onMapTaskSelectionChanged={onMapTaskSelectionChanged}
+    onPhaseSelectionChanged={onPhaseSelectionChanged}
```

Comment:
It looks like it's need to be scaled or something. 
When I just open it I don't see anything
<img width="1165" alt="Screen Shot 2022-05-21 at 10 14 47 PM" src="https://user-images.githubusercontent.com/55718143/169680073-ed0ba161-fad9-4482-853d-52665c21b1f9.png">
If I scale whole page with "-" Zoom Out icon in storybook I can see a glimpse of it.
<img width="1728" alt="Screen Shot 2022-05-21 at 10 16 36 PM" src="https://user-images.githubusercontent.com/55718143/169680088-3d18bfe1-3182-4357-9e14-4ba043032252.png">